amd_openvx/openvx/ago/ago_internal.h

```diff
@@ -629,6 +631,9 @@ struct AgoNode {
     cl_event opencl_event;
 #elif ENABLE_HIP
     hipStream_t hip_stream0;
+    std::string hip_kernel_name;
```

Comment:
same as above, all of these new variables have been removed from the ToT. sync your code with ToT and remove these variables from here.